app/controllers/importDetails/UserTypeController.scala

```diff
@@ -98,10 +98,11 @@ class UserTypeController @Inject()(identify: IdentifierAction,
       } yield mode
     }.getOrElse(false)
 
-    (cyaMode, appConfig.updateCaseEnabled) match {
-      case (true, _) => controllers.cya.routes.CheckYourAnswersController.onLoad()
-      case (false, true) => controllers.serviceEntry.routes.WhatDoYouWantToDoController.onLoad()
-      case (false, false) => controllers.serviceEntry.routes.ConfirmEORIDetailsController.onLoad()
+    (cyaMode, appConfig.updateCaseEnabled, appConfig.cancelCaseEnabled) match {
+      case (true, _, _) => controllers.cya.routes.CheckYourAnswersController.onLoad()
+      case (false, true, _) => controllers.serviceEntry.routes.WhatDoYouWantToDoController.onLoad()
+      case (false, _, true) => controllers.serviceEntry.routes.WhatDoYouWantToDoController.onLoad()
+      case (false, false, false) => controllers.serviceEntry.routes.ConfirmEORIDetailsController.onLoad()
```

Comment:
I think this could be written like
```scala
if (cyaMode) ...
else if (appConfig.updateCaseEnabled || appConfig.cancelCaseEnabled) ...
else ...
```
I don't have a strong preference but I think above is a little easier to read

Reply:
```suggestion
    (cyaMode, appConfig.updateCaseEnabled, appConfig.cancelCaseEnabled) match {
      case (true, _, _) => controllers.cya.routes.CheckYourAnswersController.onLoad()
      case (_, true, _) | (_, _, true) => controllers.serviceEntry.routes.WhatDoYouWantToDoController.onLoad()
      case _ => controllers.serviceEntry.routes.ConfirmEORIDetailsController.onLoad()
```
Another option if you wanna stick with a match